civihr_employee_portal/updates/7044.php

```diff
@@ -12,27 +12,40 @@ function civihr_employee_portal_update_7044() {
 
   $addresses = _get_personal_address_count();
   foreach ($addresses as $address) {
-    if ($address['total'] != 2) {
+    if ($address['total'] < 2) {
       continue;
     }
 
     $result = civicrm_api3('Address', 'get', [
       'location_type_id' => 'Personal',
       'contact_id' => $address['contact_id']
     ]);
-    $processed = 0;
-    foreach ($result['values'] as $contactAddress) {
-      $updateAnyAddress = $contactAddress['is_primary'] == 0 && $address['primary_count'] == 1;
-      $updateSecondAddress = $address['primary_count'] != 1 && $processed == 1;
-
-      if ($updateAnyAddress || $updateSecondAddress) {
-        $contactAddress['location_type_id'] = $locationType['id'];
-        $contactAddress['is_primary'] = 0;
-        civicrm_api3('Address', 'create', $contactAddress);
-      }
-
-      $processed++;
+
+    $addresses = array_values($result['values']);
```

Comment:
$addresses variable name is already used in the above, not a good idea to use the same name for another variable, maybe $contactAdresses is a better name

Reply:
Thanks for catching that. 🤦‍♂️